src/components/Upgrade/UpgradeOrganizationPanel.js

```diff
@@ -166,81 +168,46 @@ UpgradeOrganizationPanel.propTypes = {
   onClose: PropTypes.func.isRequired,
   repos: ReposListType,
   daoAddress: DaoAddressType.isRequired,
+  theme: PropTypes.object,
   wrapper: AragonType,
 }
 
-const AppVersion = ({
-  baseUrl,
-  contractAddress,
-  icons,
-  knownIcon,
-  name,
-  version,
-}) => {
-  const major = version.split('.')[0]
+const AppVersion = ({ repo }) => {
+  const { version } = repo
   return (
     <div
       css={`
-        display: flex;
+        display: inline-grid;
+        grid-template-columns: auto auto;
+        grid-gap: ${2 * GU}px;
         align-items: center;
-        margin: 10px 0;
+        margin: ${0.5 * GU}px 0;
       `}
     >
-      <div
-        css={`
-          width: 20px;
-          margin-right: 5px;
-        `}
-      >
-        v{major || version}
-      </div>
-      <ExternalLink
-        href={blockExplorerUrl('address', contractAddress, {
-          networkType: network.type,
-        })}
-        css={`
-          display: flex;
-          background: #daeaef;
-          border-radius: 3px;
-          align-items: center;
-          height: 22px;
-          align-items: center;
-          text-decoration: none;
-        `}
-      >
-        <AppIcon
-          app={{ baseUrl, icons }}
-          src={knownIcon || undefined}
-          radius={3}
-        />
-        <div
-          css={`
-            padding: 0 5px;
-          `}
-        >
-          {name}
-        </div>
-      </ExternalLink>
+      <div css="width: 26px;">{version}</div>
```

Comment:
Perhaps we can use grid for this instead, like the `UpgradeApp` panel, so that it'll stay aligned even if we get to versions with another digit or two

Reply:
It is already wrapped in a `grid` container, this is no longer needed but: 
![image](https://user-images.githubusercontent.com/3072458/63766722-1bfd9d00-c8cc-11e9-9e11-86fb072e2a66.png)
because `grid-gap` works after the items width the `RepoBadge`s are not aligned;

However, I can set a width for the `grid-column` of `4ch`: 
![image](https://user-images.githubusercontent.com/3072458/63766650-eeb0ef00-c8cb-11e9-810b-f70af9ef6912.png)
That gives enough room for several updates